src/Data/Form.purs

```diff
@@ -56,14 +57,20 @@ type StringRows f r =
   | r
   )
 
+type CurrencyRows f r =
+  ( placeholder :: f
+  | r
+  )
+
 type DropdownRows f r =
   ( options :: f
   , placeholder :: f
   | r
   )
 
 data Input f
-  = Dropdown (Record (SharedRows f + RequiredRows f + DropdownRows f + ()))
+  = Currency (Record (SharedRows f + RequiredRows f + CurrencyRows f + ()))
```

Comment:
I suppose we might want a `NumberRows` similar to `StringRows` for numeric validation, though that won't be relevant until my PR is done so maybe I should just add those in.

```purescript
type NumberRows f r =
  ( min :: f
  , max :: f
  | r
  )
```

Reply:
I'm down to wait. I didn't want to add fields we weren't using. It's tough to know what all is essential to things have an effect in a real form and what is waiting to be implemented when there are excess fields.

Reply:
They certainly aren't essential to the MVP, and we've already decided not to support all behaviors from the start. So we just need to be mindful when scoping out future sprint work that it's not only fields we need to add support for, but behaviors as well.

At any rate, since this is already done I'm okay with merging this PR in and then adding in validation as part of my PR. Or a new PR that's dependent on both.